info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T06:38:45.490Z"}
info: Connected to the database. {"timestamp":"2024-08-21T06:38:45.531Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T06:41:25.039Z"}
info: Connected to the database. {"timestamp":"2024-08-21T06:41:25.066Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T06:44:24.463Z"}
info: Connected to the database. {"timestamp":"2024-08-21T06:44:24.492Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:71:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:53:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:60:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T06:45:32.647Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T08:33:09.548Z"}
info: Connected to the database. {"timestamp":"2024-08-21T08:33:09.568Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:71:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:53:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:60:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T08:34:58.053Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T08:43:15.582Z"}
info: Connected to the database. {"timestamp":"2024-08-21T08:43:15.617Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T08:44:13.197Z"}
info: Connected to the database. {"timestamp":"2024-08-21T08:44:13.216Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T08:46:35.669Z"}
info: Connected to the database. {"timestamp":"2024-08-21T08:46:35.695Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:71:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T08:46:41.025Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:71:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T08:47:02.581Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T08:47:24.263Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:71:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T08:47:43.349Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:71:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T08:48:11.570Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:71:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T08:56:05.057Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:71:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T08:57:57.599Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: Registration_Cli.users index: Email_1 dup key: { Email: "hellouser@gmail.com" } 
      url - /api/v1/users/createuser 
      method - POST 
      IP - ::1
     {"timestamp":"2024-08-21T09:01:09.610Z"}
error: 
    status - 500 
    message - RangeError: Invalid status code: 11000
    at ServerResponse.writeHead (node:_http_server:352:11)
    at ServerResponse.writeHead (D:\Workspace\Fundoo_Notes\node_modules\on-headers\index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:338:8)
    at write_ (node:_http_outgoing:919:9)
    at ServerResponse.end (node:_http_outgoing:1030:5)
    at ServerResponse.send (D:\Workspace\Fundoo_Notes\node_modules\express\lib\response.js:233:10)
    at ServerResponse.json (D:\Workspace\Fundoo_Notes\node_modules\express\lib\response.js:279:15)
    at appErrorHandler (D:\Workspace\Fundoo_Notes\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Layer.handle_error (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:67:12)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at Immediate.next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Immediate._onImmediate (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:480:21) 
    url - /api/v1/users/createuser 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:01:09.962Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:02:04.094Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:02:04.117Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:06:53.492Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:06:53.532Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:07:18.902Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:07:18.926Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: Registration_Cli.users index: Email_1 dup key: { Email: "hellouser@gmail.com" } 
      url - /api/v1/users/createuser 
      method - POST 
      IP - ::1
     {"timestamp":"2024-08-21T09:07:26.834Z"}
error: 
    status - 500 
    message - RangeError: Invalid status code: 11000
    at ServerResponse.writeHead (node:_http_server:352:11)
    at ServerResponse.writeHead (D:\Workspace\Fundoo_Notes\node_modules\on-headers\index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:338:8)
    at write_ (node:_http_outgoing:919:9)
    at ServerResponse.end (node:_http_outgoing:1030:5)
    at ServerResponse.send (D:\Workspace\Fundoo_Notes\node_modules\express\lib\response.js:233:10)
    at ServerResponse.json (D:\Workspace\Fundoo_Notes\node_modules\express\lib\response.js:279:15)
    at appErrorHandler (D:\Workspace\Fundoo_Notes\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Layer.handle_error (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:67:12)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at Immediate.next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Immediate._onImmediate (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:480:21) 
    url - /api/v1/users/createuser 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:07:26.844Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:75:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:72:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:09:33.014Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:75:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:72:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:09:35.070Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:75:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:72:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:09:35.755Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:75:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:72:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:09:36.283Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:75:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:72:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:09:36.888Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:75:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:72:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:09:37.556Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:75:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:72:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:09:38.149Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:75:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:72:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:09:38.938Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:13:01.431Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:13:01.475Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:13:10.939Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:13:10.964Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:77:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:13:16.760Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:14:03.118Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:14:03.137Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:78:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:76:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:73:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:14:31.495Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:20:37.933Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:20:37.957Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:84:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:62:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:76:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:73:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:52:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:59:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:21:55.115Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:22:30.361Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:23:24.917Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:23:35.274Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:23:37.832Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:26:04.530Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:26:06.902Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:26:55.816Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:26:55.837Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:28:03.512Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:28:33.964Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:32:56.932Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:32:56.958Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:33:20.092Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:33:20.114Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:33:39.782Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:37:43.902Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:37:43.926Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:37:50.842Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:45:44.034Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:45:44.059Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:46:12.345Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:47:01.239Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:52:10.275Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:52:10.302Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:52:25.945Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:52:30.366Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:54:56.656Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:54:56.680Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:56:33.369Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:56:33.392Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T09:57:32.797Z"}
info: Connected to the database. {"timestamp":"2024-08-21T09:57:32.823Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T09:58:21.965Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:00:02.769Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:00:02.791Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:00:40.211Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:00:40.238Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-Password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T10:01:04.348Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:07:50.657Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:07:50.682Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-Password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T10:10:42.657Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:12:41.826Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:12:41.848Z"}
error: 
    status - 500 
    message - Error: User Not Found
    at _callee3$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:48:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-Password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T10:15:39.061Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:19:02.200Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:19:02.224Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:20:28.646Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:20:28.675Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:23:31.419Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:23:31.442Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:26:51.163Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:26:51.187Z"}
error: 
    status - 500 
    message - Error: Invalid or expired reset token
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:81:13)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at Object.apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at Object.resetPassword (D:\Workspace\Fundoo_Notes\src\services\/user.service.js:61:27)
    at _callee4$ (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:77:36)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at resetPassword (D:\Workspace\Fundoo_Notes\src\controllers\/user.controller.js:74:27)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at _callee$ (D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:22:7)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _next (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:17:9)
    at D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:7
    at new Promise (<anonymous>)
    at apply (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:14:12)
    at D:\Workspace\Fundoo_Notes\src\middlewares\/auth.middleware.js:29:4
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:149:13)
    at Route.dispatch (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\route.js:119:3)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:284:15
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Function.handle (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:175:3)
    at router (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:47:12)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at logger (D:\Workspace\Fundoo_Notes\node_modules\morgan\index.js:144:5)
    at Layer.handle [as handle_request] (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:328:13) 
    url - /api/v1/users/reset-Password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T10:41:49.664Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:44:40.700Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:44:40.723Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:45:25.927Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:45:25.951Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:47:58.655Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:47:58.682Z"}
error: 
      status - 11000
      message - E11000 duplicate key error collection: Registration_Cli.users index: Email_1 dup key: { Email: "india1@gmail.com" } 
      url - /api/v1/users/createuser 
      method - POST 
      IP - ::1
     {"timestamp":"2024-08-21T10:48:04.730Z"}
error: 
    status - 500 
    message - RangeError: Invalid status code: 11000
    at ServerResponse.writeHead (node:_http_server:352:11)
    at ServerResponse.writeHead (D:\Workspace\Fundoo_Notes\node_modules\on-headers\index.js:44:26)
    at ServerResponse._implicitHeader (node:_http_server:338:8)
    at write_ (node:_http_outgoing:919:9)
    at ServerResponse.end (node:_http_outgoing:1030:5)
    at ServerResponse.send (D:\Workspace\Fundoo_Notes\node_modules\express\lib\response.js:233:10)
    at ServerResponse.json (D:\Workspace\Fundoo_Notes\node_modules\express\lib\response.js:279:15)
    at appErrorHandler (D:\Workspace\Fundoo_Notes\src\middlewares\/error.middleware.js:36:26)
    at Layer.handle_error (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:71:5)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Layer.handle_error (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\layer.js:67:12)
    at trim_prefix (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:326:13)
    at D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:286:9
    at Function.process_params (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:346:12)
    at Immediate.next (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:280:10)
    at Immediate._onImmediate (D:\Workspace\Fundoo_Notes\node_modules\express\lib\router\index.js:646:15)
    at processImmediate (node:internal/timers:480:21) 
    url - /api/v1/users/createuser 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T10:48:04.803Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:50:12.175Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:50:12.215Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:51:02.902Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:51:02.930Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:51:22.364Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:51:22.389Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:52:56.401Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:52:56.422Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T10:54:23.469Z"}
info: Connected to the database. {"timestamp":"2024-08-21T10:54:23.496Z"}
info: Server started at http://http://localhost:4500/api/v1/ {"timestamp":"2024-08-21T13:16:38.097Z"}
info: Connected to the database. {"timestamp":"2024-08-21T13:16:38.119Z"}
error: 
    status - 500 
    message - Error: Failed to send reset email
    at _callee$ (D:\Workspace\Fundoo_Notes\src\utils\/sendMail.js:32:11)
    at tryCatch (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:45:16)
    at Generator.<anonymous> (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:133:17)
    at Generator.throw (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:74:21)
    at asyncGeneratorStep (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:17)
    at _throw (D:\Workspace\Fundoo_Notes\node_modules\@babel\runtime\helpers\asyncToGenerator.js:20:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/forgot-Password 
    method - POST 
    IP - ::1
   {"timestamp":"2024-08-21T13:18:16.084Z"}
